netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -631,56 +612,51 @@ _net_ebpf_ext_compare_destination_address(_In_ const bpf_sock_addr_t* addr1, _In
 }
 
 static void
-_net_ebpf_ext_connection_context_initialize_key(
+_net_ebpf_extension_connection_context_initialize(
     _In_ const bpf_sock_addr_t* sock_addr_ctx,
     uint64_t transport_endpoint_handle,
-    BOOLEAN original,
-    BOOLEAN v4_mapped,
-    _Out_ net_ebpf_ext_connection_context_key_t* context_key)
+    bool set_timestamp,
+    _Out_ net_ebpf_extension_connection_context_t* connection_context)
 {
-    // In case of original connection context, the destination IP classifiable field for classify
-    // callback at the AUTH_CONNECT_V4 layer is not populated for v4-mapped v6 address case.
-    // So for v4-mapped case, do not fill the destination IP in the key, to be able to match
-    // the context with the incoming values in AUTH callout.
-    if (!(original && v4_mapped)) {
-        RtlCopyMemory(
-            context_key->address_info.destination_ip.ipv6,
-            sock_addr_ctx->user_ip6,
-            sizeof(context_key->address_info.destination_ip));
-    }
-
-    context_key->address_info.destination_port = sock_addr_ctx->user_port;
-    context_key->address_info.family = sock_addr_ctx->family;
-    context_key->address_info.source_port = sock_addr_ctx->msg_src_port;
-    context_key->transport_endpoint_handle = transport_endpoint_handle;
-    context_key->protocol = (uint16_t)sock_addr_ctx->protocol;
-    context_key->compartment_id = sock_addr_ctx->compartment_id;
+    RtlCopyMemory(
+        connection_context->address_info.destination_ip.ipv6,
+        sock_addr_ctx->user_ip6,
+        sizeof(connection_context->address_info.destination_ip));
+
+    connection_context->address_info.destination_port = sock_addr_ctx->user_port;
+    connection_context->address_info.family = sock_addr_ctx->family;
+    connection_context->address_info.source_port = sock_addr_ctx->msg_src_port;
+    connection_context->transport_endpoint_handle = transport_endpoint_handle;
+    connection_context->protocol = (uint16_t)sock_addr_ctx->protocol;
+    connection_context->compartment_id = sock_addr_ctx->compartment_id;
+    if (set_timestamp)
+        connection_context->timestamp = CONVERT_100NS_UNITS_TO_MS(KeQueryInterruptTime());
```

Comment:
missing braces for ```if``` block, per [coding conventions](https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#formatting-notes)